janusgraph-test/src/test/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -51,35 +45,34 @@ public void graphShouldOpenWithSameInstanceId() {
         final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
 
         final StandardJanusGraph graph2 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
```

Comment:
Please change the line:
`assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");`
to
`assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");`